DP02_16_Galaxy_Cluster_Weak_Lensing.ipynb

```diff
@@ -0,0 +1,1566 @@
+{
```

Comment:
<p>Slight suggested rewording for clarity: ""The large mass of a galaxy cluster coherently distorts the images of background galaxies, and this distortion occurs over a large area of the sky around the cluster. The lensing effect on the shape of a single background galaxy far from the cluster center is small, but the lensing signal can be detected by averaging the shapes of many galaxies. This is because the original shapes of the galaxies should be randomly oriented.""</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>adjusted</p>